ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -577,6 +577,9 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.address_family_evpn.route.import_match_failure_action is arista.avd.defined('discard') %}
       route import match-failure action discard
 {%         endif %}
+{%         if router_bgp.address_family_evpn.next_hop_unchanged is arista.avd.defined(true) %}
+      bgp next-hop-unchanged
```

Comment:
Please confirm that the configuration order on EOS has this as the last command compared to the others under evpn.

Reply:
According to the output of EOS command "show cli commands all-modes verbatim | grep RouterBgpBaseAfEvpn", the "next-hop-unchanged" should be preceding all the remaining knobs of address-family evpn. Hence reordering at the J2 template level